install/kubernetes/helm/istio/charts/security/templates/configmap.yaml

```diff
@@ -11,6 +11,8 @@ metadata:
     istio: security
 data:
   custom-resources.yaml: |-
-    {{- include "security-default.yaml.tpl" . | indent 4}}
+    {{- if .Values.global.mtls.enabled }}
+      {{- include "security-default.yaml.tpl" . | indent 4}}
+    {{- end }}
```

Comment:
while this is confusing, I don't think this is causing any issue. Make sure that the run.sh stuff below still works

Reply:
I feel this need to be fixed though no scenario is broken for now(btw, integration test pass)